DuckDuckGo/BrowsingTips.swift

```diff
@@ -61,13 +61,17 @@ class BrowsingTips {
         switch tip {
             
         case .privacyGrade:
-            delegate?.showPrivacyGradeTip()
+            delegate?.showPrivacyGradeTip(didShow: didShow(shown:))
             
         case .fireButton:
-            delegate?.showFireButtonTip()
+            delegate?.showFireButtonTip(didShow: didShow(shown:))
             
         }
         
+    }
+ 
+    private func didShow(shown: Bool) {
```

Comment:
I'd make it `didShow(_ shown: Bool)` as function name seems to imply what the first parameter is.

Reply:
Agreed